docs/guide/bundles.md

```diff
@@ -24,59 +24,106 @@ To learn more about organizing your eZ Platform project, see [Best Practices](be
 
 You can generate a new bundle using a `generate:bundle` command. See [Symfony documentation on generating bundles](http://symfony.com/doc/current/bundles/SensioGeneratorBundle/commands/generate_bundle.html).
 
-#### How to remove a bundle?
+### How to remove bundles?
 
 To remove a bundle (either one you created yourself, or an out-of-the-box one that you do not need) see the [How to Remove a Bundle](http://symfony.com/doc/current/bundles/remove.html) instruction in Symfony doc.
 
 ## Built-in bundles
 
-Some of the key built-in bundles are:
-
-#### eZ Platform kernel
-
-[ezpublish-kernel](https://github.com/ezsystems/ezpublish-kernel) contains the core of the whole
-eZ Platform application.
-
-##### EzPublishCoreBundle
-
-EzPublishCoreBundle is contained in [ezpublish-kernel](https://github.com/ezsystems/ezpublish-kernel).
-
-To get an overview of EzPublishCoreBundle's configuration, run the following command-line script:
-
-``` bash
-php app/console config:dump-reference ezpublish
-```
-
-#### PlatformUIBundle
-
-[PlatformUIBundle](https://github.com/ezsystems/PlatformUIBundle) provides the main editing and back-end interface for eZ Platform.
-
-#### ezplatform-solr-search-engine
-
-[ezplatform-solr-search-engine](https://github.com/ezsystems/ezplatform-solr-search-engine)
-is the [Solr-powered](http://lucene.apache.org/solr/) search handler for eZ Platform.
-
-#### Repository Forms
-
-[Repository Forms](http://github.com/ezsystems/repository-forms) is a bundle which provides form-based interaction for the Repository Value objects.
-
-It is currently used by:
-
-- `ezsystems/platform-ui-bundle` for most management interfaces: Sections, Content Types, Roles, Policies, etc.
-- `ezsystems/ezpublish-kernel` for user registration and user generated content
-
-!!! enterprise
-
-    #### StudioUIBundle
-
-    StudioUIBundle contains the Studio editing interface
-    provided in eZ Platform Enterprise Edition.
-
-    #### Landing Page Field Type Bundle
-
-    EzLandingPageFieldTypeBundle provides
-    the Landing Page that is at the heart of StudioUI.
+The eZ Platform directory contains multiple sub-directories.
+Each sub-directory is dedicated to a specific part of the system and contains a collection of logically related files.
 
 !!! tip
 
     You can see the bundles that are automatically installed with eZ Platform in [composer.json](https://github.com/ezsystems/ezplatform/blob/master/composer.json).
+
+The following tables give an overview of the main eZ Platform directories.
+
+### Core bundles
+
+|Bundle|Description|
+|---------|-----------|
+|[ezplatform](https://github.com/ezsystems/ezplatform)|meta repository that pulls in all dependencies for clean distribution of eZ Platform|
```

Comment:
this is not actually a bundle, it's the one for he flavors of the project you get started with.

same with ezplatform-ee below.

I think these are listed on install doc.